exporter/collector/logs.go

```diff
@@ -0,0 +1,50 @@
+package collector
+
+import (
+	"context"
+	"fmt"
+
+	"cloud.google.com/go/logging"
+	"go.opentelemetry.io/collector/model/pdata"
+)
+
+type LogsExporter struct {
+	client *logging.Client
+	logger *logging.Logger
+}
+
+func NewGoogleCloudLogsExporter(ctx context.Context, cfg Config) (*LogsExporter, error) {
+	client, err := logging.NewClient(ctx, "mikedame-joonix")
```

Comment:
It looks like this should be of the form `projects/<project id>`: https://github.com/googleapis/google-cloud-go/blob/main/logging/logging.go#L126.  Can you add a TODO to detect the project id as well?